src/modules/stock/index.ts

```diff
@@ -0,0 +1,59 @@
+import api from "@/api";
+import { hasError } from "@/util";
+import { Stock, Response } from "@/types"
+import { transform } from "node-json-transform";
+import { stockTransformRule } from "@/mappings/stock";
+
+async function fetchProductsStock(productIds: Array<string>, facilityId?: string): Promise<Array<Stock> | Response> {
```

Comment:
As we are not fetching overall inventory in this method, we should rename based upon use case